GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -355,18 +357,18 @@ public override bool TryRunInstaller(InstallActionWrapper installActionWrapper,
                     foreach (InstallActionInfo entry in platformInstallManifest.InstallActions)
                     {
                         currentInstallAction = entry;
-                        string installerPath = Path.Combine(this.ExtractedInstallerPath, ContentDirectoryName, entry.InstallerRelativePath);
+                        string installerBasePath = Path.Combine(this.ExtractedInstallerPath, ContentDirectoryName);
 
                         string args = entry.Args ?? string.Empty;
 
                         // Replace tokens on args
-                        string processedArgs = NuGetUpgrader.ReplaceArgTokens(args, this.UpgradeInstanceId, ProductUpgraderInfo.GetLogDirectoryPath());
+                        string processedArgs = NuGetUpgrader.ReplaceArgTokens(args, this.UpgradeInstanceId, ProductUpgraderInfo.GetLogDirectoryPath(), $"\"{installerBasePath}\"");

```

Comment:
Could you provide some details on this change?  Why do we have a token for the installer base path now?